rpmlb/app.py

```diff
@@ -21,16 +21,18 @@ def __init__(self):
 
     def run(self, argv=None):
         LOG.info("Starting %s (%s)", self._program, self._version)
-        is_error = False
+        is_success = False
         try:
             self.run_internally(argv)
+            is_success = True
```

Comment:
Move this to the `else` part of the `try-except-else` statement. You are only trying the thing above.